src/plugins/intel_cpu/src/nodes/executors/acl/acl_mvn.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+// TODO: remove relative path
+#include "../mvn.hpp"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class AclMVNExecutor : public MVNExecutor {
+public:
+    AclMVNExecutor();
+
+    bool init(const MVNAttrs& mvnAttrs,
+              const std::vector<MemoryDescCPtr>& srcDescs,
+              const std::vector<MemoryDescCPtr>& dstDescs,
+              const dnnl::primitive_attr &attr) override;
+    void exec(const std::vector<MemoryCPtr>& src, 
+              const std::vector<MemoryPtr>& dst, 
+              const void *post_ops_data_) override;
+
+    impl_desc_type getImplType() const override {
+        return implType;
+    }
+
+private:
+    impl_desc_type implType = impl_desc_type::acl;
+
+    arm_compute::Tensor srcTensor;
+    arm_compute::Tensor dstTensor;
+    std::unique_ptr<arm_compute::NEMeanStdDevNormalizationLayer> mvn = nullptr;
+};
+
+class AclMVNExecutorBuilder : public MVNExecutorBuilder {
+public:
+    bool isSupported(const MVNAttrs& mvnAttrs, 
+                     const std::vector<MemoryDescCPtr>& srcDescs, 
+                     const std::vector<MemoryDescCPtr>& dstDescs) const override {
+        if ((srcDescs[0]->getPrecision() != InferenceEngine::Precision::FP32 &&
```

Comment:
Such comparison allows src and dst precisions to be different, like src:FP32 and dst:FP16. Is it really supported by ACL impl?
Same question about layouts.